models/issue_comment.go

```diff
@@ -40,6 +40,8 @@ const (
 	CommentTypeLabel
 	// Milestone changed
 	CommentTypeMilestone
+	// Assignees changed
+	CommentTypeAssignees
```

Comment:
Because currently Gitea only supports one assignee, but github support many. I think Gitea could support many in future. So I keep it plural for the next PR.